Harvest.Web/ClientApp/src/Util/DateHelper.ts

```diff
@@ -0,0 +1,7 @@
+export const EnsureUTC = (date: Date) => {
+  if (date.toString().includes("Z")) {
+    return new Date(date);
+  }
+
+  return new Date(date + "Z");
```

Comment:
I don't have a much better suggestion but I don't love relying on the implicit string conversion aspect here. Perhaps do like date.toISOString() or one of those methods? If not we can figure it out later, but it would nice to have something that gives a solidly defined output that won't vary between browsers.